macros/src/lib.rs

```diff
@@ -67,6 +67,10 @@ impl DerivedTS {
             #impl_start {
                 const EXPORT_TO: Option<&'static str> = Some(#export_to);
 
+                fn ident() -> String {
+                    stringify!(#rust_ty).to_owned()
+                }
+                
```

Comment:
That's not sufficient, since it doesn't account for `#[ts(rename)]`. The "real" TS ident is in `DerivedTS`, I think we should use that.